.github/workflows/shared_workflow.yml

```
@@ -92,26 +92,21 @@ jobs:
           cd server && git submodule update --init
           ./occ maintenance:install --admin-pass=admin
 
-      - name: PHP code analysis
+      - name: PHP code analysis and linting
         run: |
           # The following if block can be removed once Nextcloud no longer supports PHP 8.0
           if [ "${{matrix.phpVersion}}" -eq 8 ]; then
             # Ignoring the pact test for php 8.0
             sed -i '/<ignoreFiles>/a \ <file name="tests/lib/Service/OpenProjectAPIServiceTest.php"/>' psalm.xml
           fi
           make psalm
-
-      - name: PHP code style
-        run: composer run cs:check || ( echo 'Please run `composer run cs:fix` to format your code' && exit 1 )
+          make csfixer || ( echo 'Please run `make csfixer-fix` to format your code' && exit 1 )
```

```suggestion
          make csfixer || ( echo 'Please run `make csfixer-fix` to format your code' && exit 1 )
```

Why keep linting in the same step? For simpler debugging and maintenance, wouldn't it be better to move it to a separate step as before?

---

The readability and functionality would not be compromised. we really don't care the output if the step passes but when one of the command fails then it should be pretty similar to the previous one.
if `make psalm` fails then it will show the necessary error and the next command won't run.